test/mapnik/TestLoadLibrary.java

```diff
@@ -13,18 +13,9 @@
  */
 public class TestLoadLibrary {
 	@Test
-	public void testStaticSettings() {
```

Comment:
This test doesn't do anything now, so I've removed it. It also kept failing for me, I suspect that the test below was being run before this one, leaving Mapnik uninitialised.